packages/api/AlvTimeWebApi/Controllers/InvoiceRateController.cs

```diff
@@ -0,0 +1,42 @@
+using System;
+using System.Threading.Tasks;
+using AlvTime.Business.InvoiceRate;
+using AlvTimeWebApi.Controllers.Utils;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+
+namespace AlvTimeWebApi.Controllers;
+
+[Route("api/user")]
+[ApiController]
+public class InvoiceRateController : Controller
```

Comment:
Du kan arve fra ControllerBase i stedet for Controller. Controller legger til litt ekstra greier som kun brukes til views

Reply:
Bra tips. Fiksa det :+1: